README.md

```diff
@@ -7,12 +7,15 @@ GoodPoints is a collection of tools for compressing a distribution more effectiv
 - Given an initial summary of n input points, **kernel thinning** returns s << n output points with comparable integration error across a reproducing kernel Hilbert space
 - **Compress++** reduces the runtime of generic thinning algorithms with minimal loss in accuracy
 - **Compress Then Test** accelerates kernel two-sample testing using high-fidelity compression
+- **Debiased Distribution Compression** provides a suite of JAX-accelerated methods for debiasing and compressing input points, supporting equal-weighted, simplex-weighted, and constant-preserving coresets
```

Comment:
I would remove the Debiased Distribution Compression bullet and instead add the following bullet:
- **Stein Kernel Thinning**, **Stein Recombination**, and **Stein Cholesky** deliver unweighted,  simplex-weighted, and constant-preserving coresets that reduce biases due to off-target sampling, tempering, or burn-in